pyproject.toml

```diff
@@ -6,11 +6,90 @@ requires = [
   "wheel",
 ]
 
+[project]
+name = "compliance-checker"
+description = "Checks Datasets and SOS endpoints for standards compliance"
+readme = "README.md"
+license = {text = "Apache-2.0"}
+maintainers = [
+    {name = "Dave Foster", email = "dave@axiomdatascience.com"},
+    {name = "Benjamin Adams"},
+    {name = "Luke Campbell"},
+    {name = "Filipe Fernandes"},
```

Comment:
@benjwadams are you OK with this change? Most projects are moving away from "author" in liue for maintainers.